ui/src/app/modules/boards/pages/login/login.page.ui.spec.ts

```diff
@@ -76,22 +76,33 @@ describe('Logging in', () => {
       component = await createComponent();
     });
 
+    afterEach(() => {
+      jest.resetAllMocks();
+    });
+
     it('Displays an error message if team name is not entered', async () => {
       component.getByText('Sign In').click();
       await component.findByText('Please enter a team name');
     });
+
     it('Displays an error message if password is not entered', async () => {
-      enterTextIntoFormElement(component, 'Board name', teamName);
+      enterTextIntoFormElement(component, 'Team name', teamName);
       component.getByText('Sign In').click();
 
       await component.findByText('Please enter a password');
     });
+
     it('Displays an error message if password is entered correctly', async () => {
-      enterTextIntoFormElement(component, 'Board name', teamName);
+      console.error = jest.fn();
```

Comment:
We should probably take the console log out. There's no point to logging on the UI since nobody will ever see it.

Reply:
Out of the root code? I'm not too worried about it since it's in the angular code and it's been there forever.. I was just tired of it cluttering up the test output.

Reply:
Ah okay. I'll make a note to remove it in an issue or something then.